editor/src/core/workers/parser-printer/parser-printer-comments.spec.ts

```diff
@@ -209,18 +149,19 @@ describe('Parsing and printing code with comments', () => {
           // Comment inside inner JS
           return 2
         } // Comment after inner JS
+
         // Comment before return statement
         return (
           <div /* Comment before all attributes */
             data-uid='aaa' /* Comment inside attributes */
             someProp={/* Comment at start of JSX attribute */ 1000 /* Comment at end of JSX attribute */}
             someProp2={{
               /* Comment before object key */ someKey /* Comment after object key */:
-                /* Comment before object value */ 'someValue' /* Comment after object value */,
+                /* Comment before object value */ 'someValue' /* Comment after object separator */ /* Comment after object value */,
               someKey2: 'someValue2',
             }}
             someProp3={[
-              /* Comment before array value */ 100 /* Comment after array value */,
+              /* Comment before array value */ 100 /* Comment after array separator */ /* Comment after array value */,
```

Comment:
`/* Comment after array separator */` is before the array separator here

Reply:
Ha please see my comment above :grinning: